src/main/java/seedu/eduke8/parser/MenuParser.java

```diff
@@ -164,14 +164,14 @@ public Command parseCommand(DisplayableList topicList, String userInput) {
             return new BookmarkCommand(BOOKMARK_LIST, bookmarks);
         case COMMAND_NOTE:
             try {
-                if (commandArr[1].equalsIgnoreCase(COMMAND_NOTE_ADD) || commandArr[1]
-                        .equalsIgnoreCase(COMMAND_NOTE_DELETE) || commandArr[1].equals(COMMAND_NOTE_LIST)) {
+                if (commandArr[1].equalsIgnoreCase("add") || commandArr[1]
+                        .equalsIgnoreCase("delete") || commandArr[1].equals("list")) {
```

Comment:
Would it be possible to use String constants to remove the magic literals?

Reply:
done!